system/Common.php

```diff
@@ -1076,19 +1076,25 @@ function stringify_attributes($attributes, bool $js = false): string
 if (! function_exists('timer')) {
     /**
      * A convenience method for working with the timer.
-     * If no parameter is passed, it will return the timer instance,
-     * otherwise will start or stop the timer intelligently.
+     * If no parameter is passed, it will return the timer instance.
+     * If callable is passed, it measures time of callable and
+     * returns its return value if any.
+     * Otherwise will start or stop the timer intelligently.
      *
      * @return mixed|Timer
      */
-    function timer(?string $name = null)
+    function timer(?string $name = null, ?callable $callable = null)
     {
         $timer = Services::timer();
 
         if (empty($name)) {
             return $timer;
         }
 
+        if (null !== $callable) {
```

Comment:
I thought we had a CS Fixer rule for this but maybe not. 
```suggestion
        if ($callable !== null) {
```

Reply:
It used to be `! Is_null($callable)`, but fixer turned it into `null !== $callable`, which looked strange for me, but it was the tools suggestion.